perception/elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -156,6 +204,32 @@ void ElevationMapLoaderNode::onPointcloudMap(
   }
 }
 
+void ElevationMapLoaderNode::onPointCloudMapMetaData(
+  const autoware_map_msgs::msg::PointCloudMapMetaData pointcloud_map_metadata)
+{
+  RCLCPP_INFO(this->get_logger(), "subscribe pointcloud_map metadata");
+  {
+    if (pointcloud_map_metadata.metadata_list.size() < 1) {
+      RCLCPP_ERROR(
```

Comment:
> Isn't it possible that it could be 0 when it is not split?

No, it is not possible. This function is called only when `use_sequential_load` is true.  At that condition, `pointcloud_map_metadata.metadata_list.size()` should not be 0. 
When the user wants to get not split point cloud,  the user should set `use_sequential_load` false.